target/rspamd/local.d/settings.conf

```diff
@@ -6,7 +6,7 @@ authenticated {
     priority = high;
     authenticated = yes;
     apply {
-        groups_enabled = [];
+        groups_enabled = [dkim];
```

Comment:
TBH me neither. Here are the Rspamd docs:

"Rspamd rules are typically organized into groups, with each symbol capable of belonging to multiple groups. For instance, the DKIM_ALLOW symbol is part of both the dkim group and the policies metagroup. You have the flexibility to group or not group your own rules."

Reply:
"grouped rules" or "rule groups" sounds more appropriate then when referring to these "groups".

I see that you dropped referring to groups jargon and just highlight that DKIM signing will be unaffected, so that's fine 👍